op-program/host/kvstore/kv.go

```diff
@@ -0,0 +1,16 @@
+package kvstore
+
+import (
+	"errors"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// NotFoundErr is returned when a pre-image cannot be found in the KV store.
+var NotFoundErr = errors.New("not found")
```

Comment:
Isn't this usually `ErrNotFound`?

Reply:
You're right, the standard library prefixes errors like that. With custom error types they're often suffixed like `MyCustomError`, don't think there's a reason to do it one way or the other, but will change it to match stdlib exported-errors naming.